### DISCUSSION TITLE
Open the door to community development

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/nc-time-axis#60] [Ideas]
@SciTools/nc-time-axis-devs @SciTools/iris-devs 

To be honest, I think we're struggling to give `nc-time-axis` the love and attention that it deserves.

I believe that `nc-time-axis` still has benefit and use to the community outside SciTools, and I'd love to see the package evolve with the new capabilities and features that people desperately need and require.

That said, I believe there are a handful of willing and able community developers out there who would gladly nurture `nc-time-axis`, if given the opportunity, and help maintain it as an open-source project. Simply because it is a small but valuable part of their workflow.

If you are one such developer, then please let us know. I personally would be willing to welcome you with open arms to help maintain `nc-time-axis`.

**Comment by bjlittle** [comment]
I'm aiming to re-boot `nc-time-axis` to bring the CI back on line, and show it some infra-structure love.

In fact, I'm working on a PR right now... so this will help unblock the current open PRs and grease with wheels for tackling our open issues

**Reply by rcomer** [reply]
I think #57 already proposes a fix for the CI, though I may well have misunderstood.

**Reply by bjlittle** [reply]
I think we can bring both approaches together (that PR and my in-bound PR), but we need to get off `travis-ci` first, and there is a bunch of tech debt to address to breath live back into the package - nothing major, but essential house-keeping that we've sadly neglected to do :+1:

**Reply by ocefpaf** [reply]
I was going to offer to send a PR to improve the CIs and they I realized I already did :smile:

**Reply by bjlittle** [reply]
@ocefpaf You're that awesome, and you didn't realize it 😂

**Comment by spencerkclark** [comment]
I would be happy to pitch in to help maintain `nc-time-axis`.  I appreciate the work that you have put into it over the years.  It's a great tool that addresses an important need in our community.

**Reply by bjlittle** [reply]
Hey @spencerkclark!

Thanks for joining the discussion :tada: 

I've already sent you an invite to join @SciTools/nc-time-axis-devs :beers: 

Let's get this wagon rollin'...

**Reply by spencerkclark** [reply]
Awesome, I just joined -- I'm looking forward to helping get past these recent compatibility issues with `cftime` and making sure things continue to work smoothly in the future!  Thanks for getting started on updating the CI.

**Comment by aulemahal** [comment]
Nice! I am willing to help too! This package covers a small but essential part of many workflows. I agree on the needed changes to the CI!

**Comment by bjlittle** [comment]
Welcome aboard @spencerkclark and @aulemahal! :partying_face:

**Reply by bjlittle** [reply]
I'm going to create repo labels akin to SciTools/iris and a milestone for a `nc-time-axis` 1.3.0 release

I've also enabled projects for the repo, so I'll create a kanban board to coordinate the effort for the 1.3.0 release.

We can load the project board with the minimum tickets required to re-boot `nc-time-axis` with a minor release, and roll from there.

Does that sound like a reasonable plan?

**Reply by aulemahal** [reply]
Sounds quite reasonable, Thanks!

**Reply by spencerkclark** [reply]
Yes, I agree!
